<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>701</width>
    <height>476</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,5">
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="sizeConstraint">
         <enum>QLayout::SetDefaultConstraint</enum>
        </property>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="MyGLWidget" name="myGLWidget" native="true">
          <property name="font">
           <font>
            <family>Kozuka Gothic Pr6N H</family>
            <pointsize>12</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <widget class="QLabel" name="camFrame">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>160</width>
             <height>120</height>
            </rect>
           </property>
           <property name="text">
            <string>Image</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
          <widget class="QProgressBar" name="progressBar">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>120</y>
             <width>141</width>
             <height>31</height>
            </rect>
           </property>
           <property name="autoFillBackground">
            <bool>true</bool>
           </property>
           <property name="value">
            <number>50</number>
           </property>
          </widget>
          <widget class="QLabel" name="labelChrono">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>210</y>
             <width>161</width>
             <height>16</height>
            </rect>
           </property>
           <property name="autoFillBackground">
            <bool>true</bool>
           </property>
           <property name="text">
            <string>00:00:00</string>
           </property>
          </widget>
          <widget class="QLabel" name="labelCible">
           <property name="geometry">
            <rect>
             <x>160</x>
             <y>10</y>
             <width>211</width>
             <height>16</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="autoFillBackground">
            <bool>true</bool>
           </property>
           <property name="text">
            <string>nombre de cibles restantes :</string>
           </property>
          </widget>
          <widget class="QLabel" name="labelChrono2">
           <property name="geometry">
            <rect>
             <x>490</x>
             <y>90</y>
             <width>171</width>
             <height>21</height>
            </rect>
           </property>
           <property name="autoFillBackground">
            <bool>true</bool>
           </property>
           <property name="text">
            <string>00:00:00</string>
           </property>
          </widget>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="zoomSlider">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>100</number>
          </property>
          <property name="value">
           <number>100</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="tickPosition">
           <enum>QSlider::TicksBelow</enum>
          </property>
          <property name="tickInterval">
           <number>1</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="SliderAngleCatapulte">
          <property name="minimum">
           <number>-45</number>
          </property>
          <property name="maximum">
           <number>45</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="tickPosition">
           <enum>QSlider::TicksBelow</enum>
          </property>
          <property name="tickInterval">
           <number>1</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="SliderAngleBras">
          <property name="minimum">
           <number>-135</number>
          </property>
          <property name="maximum">
           <number>30</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="tickPosition">
           <enum>QSlider::TicksBelow</enum>
          </property>
          <property name="tickInterval">
           <number>1</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,0,0">
      <property name="spacing">
       <number>6</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="text">
         <string>Lauch Ball</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="boutonPlay">
        <property name="text">
         <string>Play</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBox">
        <property name="text">
         <string>Start Tracking</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>701</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>MyGLWidget</class>
   <extends>QWidget</extends>
   <header location="global">myglwidget.h</header>
   <container>1</container>
   <slots>
    <slot>setXRotation(int)</slot>
    <slot>setZRotation(int)</slot>
    <slot>setYRotation(int)</slot>
    <slot>setZoom(int)</slot>
    <slot>setAngleCatapulte(int)</slot>
    <slot>setAngleBras(int)</slot>
    <slot>launchBall()</slot>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>zoomSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>myGLWidget</receiver>
   <slot>setZoom(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>522</x>
     <y>382</y>
    </hint>
    <hint type="destinationlabel">
     <x>522</x>
     <y>203</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>SliderAngleCatapulte</sender>
   <signal>valueChanged(int)</signal>
   <receiver>myGLWidget</receiver>
   <slot>setAngleCatapulte(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>354</x>
     <y>382</y>
    </hint>
    <hint type="destinationlabel">
     <x>354</x>
     <y>186</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>SliderAngleBras</sender>
   <signal>valueChanged(int)</signal>
   <receiver>myGLWidget</receiver>
   <slot>setAngleBras(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>354</x>
     <y>431</y>
    </hint>
    <hint type="destinationlabel">
     <x>354</x>
     <y>194</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>myGLWidget</receiver>
   <slot>launchBall()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>350</x>
     <y>435</y>
    </hint>
    <hint type="destinationlabel">
     <x>354</x>
     <y>168</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
